osx/create-osx-bundle.sh

```diff
@@ -26,8 +26,8 @@ EXTRAS="base"
 PYVER="3.9.9"
 OPENSSL_VERSION="1.1.1l"
 SDL_VERSION="release-2.0.20"
-SDL_IMAGE_VERSION="168ceb577c245c91801c1bcaf970ef31c9b4d7ba"
-SDL_MIXER_VERSION="64120a41f62310a8be9bb97116e15a95a892e39d"
+SDL_IMAGE_VERSION="release-2.6.0"
+SDL_MIXER_VERSION="release-2.6.1"
```

Comment:
Can we target the dependencies update into a separate PR? (We need to keep all the SDL versions in sync with the ones we use on the CI for `kivy/kivy`) [Unless there's an issue which only affects the .App artifact]

Reply:
compiled kivy.app and fine
but if nedd revert